pages/pipelines/command_step.md.erb

```diff
@@ -241,8 +241,36 @@ steps:
 ```
 {: codeblock-file="pipeline.yml"}
 
+If you retry a job, the information about the failed job(s) remains, and a new job is created. The history of retried jobs is preserved and immutable.
 
-## Automatic retry attributes
+<%= image "retry-time-date.png", width: 2304/2, height: 698/2, alt: "You can view how and when a job was retried" %>
+
+You can also see when a job has been retried and whether it was retried automatically or by a user. The number of retries is available as an environment variable on the job.
```

Comment:
I feel this is slightly unclear. Do you mean that if you have 

1. original job
2. first retry 
3. second retry

the jobs would have retry 0,1,2  respectively? And we should definitely mention what environment variable it is.

Reply:
P.S. Here is a screenshot that's pretty explicit about it. Might be better to use it, after all. https://github.com/buildkite/docs/pull/1481#issuecomment-1117575872

Reply:
Yes!